requirements-dev.in

```diff
@@ -0,0 +1,7 @@
+pytest
```

Comment:
On n'inclut pas les dépendances déclarées dans le _setup.py_ ?

Reply:
J'aime bien le faire, mais ça nécessite deux étapes : d'abord épingler le setup.py dans un `requirements.txt` (a coup de `pip-compile setup.py`) puis dépendre du `requirements.txt` dans le `requirements-dev.in`.

Je le fais sur certains projets à moi mais je trouve ça un peu alambiqué et je n'ai pas osé le faire ici, mais c'est peut être quand même une bonne idée, on essaye ?

Sinon j'ai tenté de dépendre de `.` dans `requirements-dev.in`, ce serait un bon raccourci, mais il "matérialise" le `.` en `/home/mdk/.../...` dans le `requirements-dev.txt` donc bof bof bof :p

Reply:
:+1: 

Ça ne me semble pas _alambiqué_. Quand je lance les tests depuis mon IDE (pas avec tox donc) sur un venv vierge, ça ne marche pas parce que requirements-dev.txt ne contient pas gitignore-parser.

J'essaye.

Reply:
On dirait qu'on a deja un requirements.txt, donc on peut dépendre de `requirements.txt` dans `requirements-dev.in`. Pendant que tu y est, autogénère `requirements.txt` a coup de `pip-compile setup.py` :)

Reply:
https://github.com/AFPy/Potodo/pull/112/files